src/createResizePanel.ts

```diff
@@ -24,14 +26,20 @@ export interface ResizePanelMixin {
 		handle: string;
 	};
 	width: string;
+	height: string;
 }
 
 export type ResizePanel = Widget<ResizePanelState> & ParentListMixin<Child> & Destroyable & ResizePanelMixin;
 
 export interface ResizePanelFactory extends ComposeFactory<ResizePanel, ResizePanelOptions> { }
 
 const resizeNodePropertiesMap = new WeakMap<ResizePanel, VNodeProperties>();
-const resizingMap = new WeakMap<ResizePanel, { width: number, clientX: number }>();
+const resizingMap = new WeakMap<ResizePanel, {
```

Comment:
Yeah, but should be dealt with as a separate issue.  I think there is other private state here that should be rationalised into a single weakmap and interface.